posts/2024-06-28-gno-debugger/README.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Debugging Gno Programs"
+publication_date: 2024-06-28T13:37:00Z
+slug: gno-debugger-blog-post
+tags: [blog, post, tutorial, gno, debugger]
+authors: [@mvertes]
+---
+# Debugging Gno Programs
+ 
+In this article, we introduce the new Gno debugger feature and show how it can be used to better understand gno programs and help fixing bugs.
+
+## Motivation for a Gno debugger
+
+> Debugging is twice as hard as writing code.
+> Brian Kerninghan, "The Elements of Programming Style"
+
+> On average, you spend about eight to ten times debugging as you do writing code.
+> Anonymous
+
+Having a good debugger is important. But the Gno language is almost Go, and gnoland itself is entirely written in Go. Could I just use the existing Go tools, i.e.  the [delve] debugger, to take control and debug my gno programs?
```

Comment:
Let's use the following forms to refer to our products:
- gno.land
- Gno (capitalized)
- GnoVM

This style was agreed upon internally

```suggestion
Having a good debugger is important. But the Gno language is almost Go, and gno.land itself is entirely written in Go. Could I just use the existing Go tools, i.e.  the [delve] debugger, to take control and debug my Gno programs?
```